docs/components/custom/SectionHeading/SectionHeadingRenderer.jsx

```diff
@@ -2,21 +2,25 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import Heading from '../../../../packages/Heading/Heading'
-import Flexbox from '../../../../shared/components/Flexbox/Flexbox'
+import FlexGrid from '../../../../packages/FlexGrid/FlexGrid'
 import Box from '../../../../packages/Box/Box'
 
-import styles from './SectionHeadingRenderer.modules.scss'
-
 const SectionHeadingRenderer = ({ id, depth, toolbar, children }) => {
   const level = Math.min(depth, 4)
 
   return (
     <Box below={3}>
       <Heading level={`h${level}`} id={id}>
-        <Flexbox direction="row">
-          {children}
-          <div className={styles.toolbar}>{toolbar}</div>
-        </Flexbox>
+        <FlexGrid limitWidth={false} gutter={false}>
```

Comment:
`div`s are not allowed inside heading tags according to html validator: https://validator.w3.org/nu/#textarea

I was fixing this in another PR (#1235 ) by doing this:

```
    <Box inline between="space-between" below={3} style={{ alignItems: 'center' }}>
      <Heading level={`h${level}`} id={id}>
        {children}
      </Heading>
      <div>{toolbar}</div>
    </Box>
```